### PR TITLE
add ReadOnlyEntryProcessor to check if entity set contains flag + Ent…

### PR DESCRIPTION
**Pull request by katherinebernstein**
…itySetService function to perform the check

### CODE REVIEW

File: src/test/java/com/openlattice/hazelcast/serializers/EntitySetContainsFlagEntryProcessorStreamSerializerTest.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.openlattice.hazelcast.serializers
+
+import com.kryptnostic.rhizome.hazelcast.serializers.AbstractStreamSerializerTest
+import com.openlattice.assembler.processors.EntitySetContainsFlagEntryProcessor
+import com.openlattice.mapstores.TestDataFactory
+
+class EntitySetContainsFlagEntryProcessorStreamSerializerTest : AbstractStreamSerializerTest<EntitySetContainsFlagEntryProcessorStreamSerializer, EntitySetContainsFlagEntryProcessor>() {
```
Comment (makosblade) [inline src/test/java/com/openlattice/hazelcast/serializers/EntitySetContainsFlagEntryProcessorStreamSerializerTest.kt]: Can just make your StreamSerializer `TestableSelfRegisteringStreamSerializer` and just get rid of this :D

File: src/main/kotlin/com/openlattice/assembler/processors/EntitySetContainsFlagEntryProcessor.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.openlattice.assembler.processors
+
+import com.openlattice.edm.EntitySet
+import com.openlattice.edm.set.EntitySetFlag
+import com.openlattice.rhizome.hazelcast.entryprocessors.AbstractReadOnlyRhizomeEntryProcessor
+import java.util.*
+
+data class EntitySetContainsFlagEntryProcessor(val flag: EntitySetFlag)
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/assembler/processors/EntitySetContainsFlagEntryProcessor.kt]: Any reason why this isn't also Offloadable?

Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/assembler/processors/EntitySetContainsFlagEntryProcessor.kt]: it is now!

**Comment by makosblade** [comment]
> Quick question about making it offloadable.

How did I not know about Offloadable until now 🤦‍♂ 😢

File: src/main/kotlin/com/openlattice/assembler/processors/EntitySetContainsFlagEntryProcessor.kt (hunk 2 above)
Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/assembler/processors/EntitySetContainsFlagEntryProcessor.kt]: ......except apparently the push failed. Now it is actually

**Pull request merged**